chcemvediet/apps/obligees/tests/test_views.py

```diff
@@ -76,10 +71,62 @@ def test_allowed_http_methods(self):
         allowed = [u'HEAD', u'GET']
         self.assert_allowed_http_methods(allowed, reverse(u'obligees:autocomplete'))
 
-    @unittest.skip(u'FIXME')
     def test_autocomplete_returns_json_with_correct_structure(self):
-        oblg1 = self._create_obligee(name=u'Agency', street=u'Westend', city=u'Winterfield', zip=u'12345', emails=u'agency@a.com')
-        oblg2 = self._create_obligee(name=u'Ministry', street=u'Eastend', city=u'Springfield', zip=u'12345', emails=u'ministry@a.com')
+        Obligee.objects.all().delete()
```

Comment:
@martinmacko47 _pre_setup od ChcemvedietTestCaseMixin sa spusti zakazdym pred spustenim jednotliveho testu, cize vzdy mi vytvorii aj default instanciu Obligee, ktora je v tychto testoch neziaduca. 
Moznosti riesenia ktore mi napadli:
a) `Obligee.objects.all().delete()`
b) `self.assertIn(oblg1, data); self.assertIn(oblg2, data); self.assertEqual(len(data), 3)`
c) odstranenie default Obligee z pomocnej premennej `data`
d) `response = self.client.get(reverse(u'obligees:autocomplete') + u'?term=ministry')` * to vsak nebude mozne pri `test_autocomplete_without_term_returns_everything`
Ku ktorej moznosti sa priklanas najviac? Za mna **b)** varianta. Alebo ti napada lepsie riesenie.

Reply:
@viliambalaz Ked je nejaky test, ktory naozaj vyzaduje, aby tam neboli ine obligees, nez tie, ktore test sam vytvori, tak vtedy ano, vtedy existujuce obligees zmazat treba.

Ale tu imho staci b). S tym, ze na zaciatku este treba `assertIsInstance` na overenie, ze `data` je pole. Posledne `assertEqual(len(data), 3)` IMHO netreba, kedze sa nevieme spolahnut, kolko tych obligees je vytovrenych.

Reply:
Thread presunuty do druheho PR: https://github.com/martinmacko47/chcemvediet/pull/318/files#r513351804